src/includes/getting-started-install/unity.mdx

```diff
@@ -1,10 +1,10 @@
-Install the SDK from [`Unity UPM`](https://docs.unity3d.com/Manual/upm-ui.html) using a Git URL to Sentry's SDK repository:
+Install the package via the [Unity Package Manager](https://docs.unity3d.com/Manual/upm-ui.html) using a Git URL to Sentry's SDK repository:
 
 ```
 https://github.com/getsentry/unity.git#{{ packages.version('sentry.dotnet.unity', '0.0.5') }}
 ```
 <Note>
 
-Confirm there's no empty space at the end of the line. The Unity Package Manager will fail to find the package if an empty space is appended to the end of the URL.
+Confirm there is no trailing whitespace at the end of the URL. The Unity Package Manager will fail to find the package otherwise.
```

Comment:
We've had a few issues with this, so we should keep this statement pretty explicit.
```suggestion
Confirm the URL doesn't have a trailing whitespace at the end. The Unity Package Manager will fail to find the package if a trailing whitespace is appended.
```